notebooks/notebooks/iclr2022/win-rate-vs-visits.ipynb

```diff
@@ -20,6 +20,7 @@
     "import matplotlib.pyplot as plt\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #75.                    [100 * splusplus_point - 100 * splusplus_point_lo],
</pre><p>Seems more natural to write this as <code>100 * (splusplus_pont - splusplus_point_lo)</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Changed. I wrote it the way I did before cause that's what copilot suggested haha.</p>